geopyspark/geotrellis/tms.py

```diff
@@ -0,0 +1,56 @@
+from PIL import Image
+import numpy as np
+import io
+
+from py4j.clientserver import ClientServer, JavaParameters, PythonParameters
+
+# Return URL of tile server for a layer
+## http://localhost:2342/s3/azavea-datahub/catalog/tms/{z}/{x}/{y}
+def layer_tms(rdd_type, uri, layer_name, render_function=None, cmap=None):
```

Comment:
What is the point of this function? It just `pass`es, and I don't see it being used anywhere where it's being implemented.